src/utils/logpointsUtil.ts

```diff
@@ -0,0 +1,23 @@
+import { setTimeout } from "timers";
+
+// tslint:disable-next-line:export-name
+// tslint:disable-next-line:no-any
+export function callWithTimeout<T>(proc: (...args: any[]) => Promise<T>, timeout: number = 60000): Promise<T> {
```

Comment:
Are we really defaulting to a 60-second timeout?!  That seems like way too long.

Reply:
Will change to 20 secs.

Reply:
Alternatively, since no timeout value is correct for all uses, you coul djust remove the default and force callers to be explicit w/ what they thing.

Reply:
@mike-kaufman Sounds good. Done that too.